ansible/roles/ansible-horizon-plugin/templates/sync_static.py

```diff
@@ -1,11 +1,10 @@
 import settings
```

Comment:
as file's name is sync_static.py and we are removing the very same which does it, Then what's the purpose of this file?

Reply:
it basically now returns a string that holds all the plugins list which are currently installed with the horizon.
It's a legacy code but not sure if any different approach is now available.